### PR TITLE
Dev rba

### PR DESCRIPTION
**Pull request by viv9k**
### Functionality:
Added rba mechanism

### Solution:
Added rba support

### Risk level:
- [x] high 
- [ ] medium
- [ ] low

### How to test:
<-- please tell us how to test functionality/bug -->

### CODE REVIEW

File: src/app/body/profile/edit-profile/edit-profile.component.html [hunk 1]
```diff
@@ -41,6 +41,20 @@ <h2>Edit Profile</h2>
...
+                                  <path d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0zm-3.97-3.03a.75.75 0 0 0-1.08.022L7.477 9.417 5.384 7.323a.75.75 0 0 0-1.06 1.06L6.97 11.03a.75.75 0 0 0 1.079-.02l3.992-4.99a.75.75 0 0 0-.01-1.05z"/>
+                                </symbol>
+                            </svg>
+                            <div class="alert alert-success d-flex align-items-center" role="alert">
+                                <svg class="bi flex-shrink-0 me-2" width="24" height="24" role="img" aria-label="Success:"><use xlink:href="#check-circle-fill"/></svg>
+                                <div>
+                                    Username available
+                                </div>
+                            </div>
+                        </a>
```
Comment (viv9k) [inline src/app/body/profile/edit-profile/edit-profile.component.html:56]: why to use svg here

File: src/app/services/RBA/rba.service.ts [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  userEmail: string = "";
+  constructor(public authService: AuthService, private functions: AngularFireFunctions ) {
+  } 
+  setDefaultPermissions(){
+    this.permissions= defaultPermissions as Permissions;
+  }
+  assignPermissions(){
+
+    if(this.isAdmin){
+      console.log("C")
```
Comment (viv9k) [inline src/app/services/RBA/rba.service.ts:26]: remove console.log

File: src/app/services/RBA/rba.service.ts [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+      this.permissions.viewMemberRoles = true;
+      this.permissions.editMemberRoles = true;
+      this.permissions.addDefaultLabel= true;
+      this.permissions.createNewLabel= true;
+      this.permissions.editLabel= true;
+      this.permissions.deleteLabel= true;
+      }
+    
+    else if(this.teamManager){
+      console.log("B");
```
Comment (viv9k) [inline src/app/services/RBA/rba.service.ts:47]: remove console.log

File: src/app/body/profile/edit-profile/edit-profile.component.html (hunk 1 above)
Comment (abhishek-mishra019) [reply src/app/body/profile/edit-profile/edit-profile.component.html:56]: it is a green tick here if username is available

**Pull request merged**